digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectService.scala

```diff
@@ -1399,7 +1399,7 @@ class ProjectService(roadAddressService: RoadAddressService, roadLinkService: Ro
           Seq(RoadAddress(NewRoadAddress, pl.roadNumber, pl.roadPartNumber, pl.roadType, pl.track, pl.discontinuity,
             pl.startAddrMValue, pl.endAddrMValue, Some(project.startDate), None, Some(project.createdBy), 0L, pl.linkId,
             pl.startMValue, pl.endMValue, pl.sideCode, pl.linkGeometryTimeStamp, pl.calibrationPoints, floating = false,
-            pl.geometry, pl.linkGeomSource, pl.ely, terminated = NoTermination))
+            pl.geometry, pl.linkGeomSource, pl.ely, terminated = NoTermination, 0))
```

Comment:
Should NewCommonHistoryId be used here instead of the zero?

Reply:
ure completely right!